include/snowflake/client.h

```diff
@@ -57,9 +57,9 @@ extern "C" {
 #define SF_COMMAND_LEN 10
 
 /**
- * The maximum object size
+ * The default maximum object size
  */
-#define SF_MAX_OBJECT_SIZE 16777216
+#define SF_DEFAULT_MAX_OBJECT_SIZE 16777216
```

Comment:
please deprecate the old const and provide the new one

Reply:
done